apps/src/lab2/views/dialogs/GenericPrompt.tsx

```diff
@@ -55,7 +55,7 @@ const GenericPromptBody: React.FunctionComponent<GenericPromptBodyProps> = ({
         value={prompt}
         onChange={e => handleInputChange(e.target.value)}
         errorMessage={errorMessage}
-        color={theme === 'Dark' ? 'white' : undefined}
+        color={theme === 'Dark' ? 'black' : undefined}
```

Comment:
can this just be `black` now?

Reply:
Or do we even need it?

Reply:
I don't think we need it! Looking at `TextField`, the default color is 'black'. 

https://github.com/code-dot-org/code-dot-org/blob/94346541e651a0e0b2d29b738b176e0ad48ff68b/frontend/packages/component-library/src/textField/TextField.tsx#L83

So I'll go ahead and remove. Thanks!

Reply:
Just noticing that the background color of the text field is now black instead of white. Confirming this with @markabarrett https://codedotorg.slack.com/archives/C06JELCAPT9/p1746050482503909